src/object/path.cpp

```diff
@@ -231,4 +231,14 @@ Path::is_valid() const
   return !m_nodes.empty();
 }
 
+void
+Path::on_flip(float height)
+{
+  for (auto& nod : m_nodes) {
+    nod.position.y = height - nod.position.y;
```

Comment:
Was there a reason why the variable isn't named `node`?

Reply:
I named it `nod` because that is the name used in `Path::move_by()`. It is weird, I will change it to `node`.